src/Pages/Repositories/ContentListTable/components/DeleteContentModal/DeleteContentModal.tsx

```diff
@@ -250,6 +259,20 @@ export default function DeleteContentModal() {
                                 </ListItem>
                               ))}
                           </ExpandableSection>
+                          <ExpandableSectionToggle
+                            isExpanded={expandState[index]}
+                            onToggle={() =>
+                                setExpandState((prev) => ({ ...prev, [index]: !prev[index] }))
+                                }
+                            toggleId='detached-expandable-section-toggle'
+                            contentId='detached-expandable-section-content'
```

Comment:
This toggleId and contentId is duplicated. 
You may want an "_up"/"_down" on the ids here.

Reply:
pushed up fixes for this and the comments below